variables.tf

```diff
@@ -247,6 +247,12 @@ variable "key_protect_use_private_endpoint" {
   default     = true
 }
 
+variable "kms_cross_account_id" {
+  type        = string
+  description = "Id of the account that owns the KMS instance to encrypt the cluster"
+  default     = null
+}
```

Comment:
suggest just calling it `kms_account_id` - mention that its only required if the KMS instance is in another account

Reply:
Done.